lib/devices/android/android.js

```diff
@@ -557,6 +558,19 @@ Android.prototype.waitForActivityToStop = function (cb) {
   this.adb.waitForNotActivity(this.args.appWaitPackage, this.args.appWaitActivity, cb);
 };
 
+Android.prototype.setActualCapabilities = function (cb) {
+  this.capabilities.deviceName = this.adb.udid || this.adb.curDeviceId;
+  this.adb.shell("getprop ro.build.version.release", function (err, version) {
+    if (err) {
+      logger.warn(err);
+      cb(err);
```

Comment:
I'm not sure this is a serious enough error to halt the session start (which is what would happen). Maybe we just want to `cb()` in either case? (In which case, it could be pulled out of the conditional)